Runtime/Tracking/Follow/Modifier/Property/Rotation/TransformPositionDifferenceRotation.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace Zinnia.Tracking.Follow.Modifier.Property.Rotation
+{
+    using UnityEngine;
+    using Malimbe.XmlDocumentationAttribute;
+    using Malimbe.PropertyValidationMethod;
+    using Malimbe.PropertySerializationAttribute;
+    using Zinnia.Extension;
+    using Zinnia.Data.Type;
+
+    /// <summary>
+    /// Updates the transform rotation of the target to match the difference in position of the source.
+    /// </summary>
+    public class TransformPositionDifferenceRotation : PropertyModifier
+    {
+        /// <summary>
+        /// The drag applied to the rotation.
+        /// </summary>
+        [Serialized, Validated]
+        [field: DocumentedByXml]
+        public float AngularDrag { get; set; } = 1f;
+
+        /// <summary>
+        /// Determines which axes to rotate.
+        /// </summary>
+        [Serialized, Validated]
+        [field: DocumentedByXml]
+        public Vector3State FollowOnAxis { get; set; } = Vector3State.True;
+
+        /// <summary>
+        /// The current angular velocity the rotation is applying to the target.
+        /// </summary>
+        public Vector3 AngularVelocity { get; protected set; }
+        /// <summary>
+        /// The previous source world position.
+        /// </summary>
+        protected Vector3? previousSourcePosition;
+
+        /// <summary>
+        /// Resets the state of the source previous position.
+        /// </summary>
+        public virtual void ResetPreviousState()
+        {
+            previousSourcePosition = null;
+        }
+
+        /// <summary>
+        /// Rotates the target based on the position difference of the source.
+        /// </summary>
+        /// <param name="source">The source to utilize in the modification.</param>
+        /// <param name="target">The target to modify.</param>
+        /// <param name="offset">The offset of the target against the source when modifying.</param>
+        protected override void DoModify(GameObject source, GameObject target, GameObject offset = null)
+        {
+            Vector3 sourcePosition = source.transform.position;
+            Vector3 targetPosition = target.transform.position;
+
+            if (previousSourcePosition == null)
+            {
+                previousSourcePosition = sourcePosition;
+            }
+
+            float xDegree = FollowOnAxis.xState ? CalculateAngle(target.transform.right, targetPosition, (Vector3)previousSourcePosition, sourcePosition) : 0f;
+            float yDegree = FollowOnAxis.yState ? CalculateAngle(target.transform.up, targetPosition, (Vector3)previousSourcePosition, sourcePosition) : 0f;
+            float zDegree = FollowOnAxis.zState ? CalculateAngle(target.transform.forward, targetPosition, (Vector3)previousSourcePosition, sourcePosition) : 0f;
+
+            AngularVelocity = new Vector3(xDegree, yDegree, zDegree) * (1f / AngularDrag);
```

Comment:
I suggest to move the `* (1f / AngularDrag)` part of this into its own method. This way subclasses can change the way the drag works, without having to replicated this whole method.

The method could be
```c#
protected virtual Vector3 ApplyDrag(Vector3 angularVelocity)
{
    return angularVelocity * (1f / AngularDrag);
}
```
perhaps (plus XML documentation comments obviously).